README.md

```diff
@@ -3,13 +3,37 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+Asymptotic analysis may be misleading with respect to actual performance in practice, due to the fact it reduces information by design. Asymptotic analysis focuses on te growth rate as the input size approaches infinity. This means naturally that more granular information about an algorithm's performance will be lost. For example an analysis such as `O(1n^2 + 2n + 3)` simplifies to `O(n^2)` since lower order terms are ignored. Most of the time this doesn't mean much, but it still means information is being lost.
+
+Another aspect that could be misleading to actual performance is the fact of unpredictable input data in the real world. Asymptotic analysis assumes worst, average, or best case scenarios for inputs relative to performance. Real world data can be skewed and produce distributions in the performance of an algorithm that have nothing to do with its raw performance.
```

Comment:
Not sure what you mean here -- if the wrong case is analyzed that would be because the analyst has misidentified it.